src/js/common/schemaform/validation.js

```diff
@@ -87,17 +87,33 @@ export function transformErrors(errors, uiSchema) {
  * This pulls custom validations specified in the uiSchema and validates the formData
  * against them.
  *
- * Expects validations that look like:
+ * Expects validations that EITHER look like:
```

Comment:
I didn't end up needing to use a custom validator, but it's the approach I looked into first. I was somewhat frustrated by the fact that I didn't realize there were two possible ways of creating a custom validation, so decided to update the documentation even though I didn't end up using a custom validator.

Reply:
Did you see the documentation concerning validation in [the formConfig document](https://github.com/department-of-veterans-affairs/vets-website/blob/master/docs/schemaform/form-config.md#i-need-to-write-custom-validation)?

Reply:
I...have never seen this documentation. Only the the [walkthrough](https://github.com/department-of-veterans-affairs/vets-website/blob/master/docs/schemaform/walkthrough.md), though I'm not sure why, since they're both in the same directory.